shared/constants/fs.js

```diff
@@ -23,58 +27,17 @@ export const makeState: I.RecordFactory<Types._State> = I.Record({
         children: I.List(['private', 'public', 'team']),
       }),
     ],
-    [
-      Types.stringToPath('/keybase/private'),
-      makeFolder({
-        children: I.List(['foo', 'foo,bar']),
-      }),
-    ],
-    [
-      Types.stringToPath('/keybase/public'),
-      makeFolder({
-        children: I.List(['foo', 'bar']),
-      }),
-    ],
-    [
-      Types.stringToPath('/keybase/team'),
-      makeFolder({
-        children: I.List(['foobar']),
-      }),
-    ],
-    [
-      Types.stringToPath('/keybase/private/foo'),
-      makeFolder({
-        children: I.List(['foo.priv']),
-      }),
-    ],
-    [
-      Types.stringToPath('/keybase/private/foo,bar'),
-      makeFolder({
-        children: I.List(['foo.bar.priv']),
-      }),
-    ],
-    [
-      Types.stringToPath('/keybase/public/foo'),
-      makeFolder({
-        children: I.List(['foo.pub']),
-      }),
-    ],
-    [
-      Types.stringToPath('/keybase/public/bar'),
-      makeFolder({
-        children: I.List(['bar.pub']),
-      }),
-    ],
-    [
-      Types.stringToPath('/keybase/team/foobar'),
-      makeFolder({
-        children: I.List(['foobar.team']),
-      }),
-    ],
-    [Types.stringToPath('/keybase/private/foo/foo.priv'), makeFile()],
-    [Types.stringToPath('/keybase/private/foo,bar/foo.bar.priv'), makeFile()],
-    [Types.stringToPath('/keybase/public/foo/foo.pub'), makeFile()],
-    [Types.stringToPath('/keybase/public/bar/bar.pub'), makeFile()],
-    [Types.stringToPath('/keybase/team/foobar/foobar.team'), makeFile()],
+    [Types.stringToPath('/keybase/private'), makeFolder()],
+    [Types.stringToPath('/keybase/public'), makeFolder()],
+    [Types.stringToPath('/keybase/team'), makeFolder()],
   ]),
 })
+
+export const makeUUID = () => uuidv1(null, Buffer.alloc(16), 0)
+export const wrapOpID = (actionCreator: Function, payload: any) =>
+  actionCreator({
+    opID: makeUUID(),
+    ...payload,
+  })
+export const dispatchFolderListLoad = (dispatch: Dispatch, path: Types.Path) =>
```

Comment:
we tend to not have any action creators anymore. if you need opIDs created on our side i'd make them in the saga and not during the action creation phase